core/src/test/java/org/infinispan/metrics/ClusteredCacheManagerMetricsTest.java

```diff
@@ -21,14 +18,13 @@
 import org.testng.annotations.Test;
 
 import io.micrometer.core.instrument.Gauge;
-import io.micrometer.core.instrument.Meter;
-import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tag;
 
 /**
  * @author anistor@redhat.com
  * @since 10.1
  */
+@Deprecated(forRemoval=true, since = "15.2")
```

Comment:
The test needs to be kept and adapted to the new format in the future.

Reply:
Removed.